neonvm/runner/main.go

```diff
@@ -703,16 +728,31 @@ func main() {
 		"-device", "virtserialport,chardev=log,name=tech.neon.log.0",
 	}
 
+	diskIdxCnt := 0
+
+	attachDisk := func(qemuCmd []string, id string, file string, extraOptions string, deviceExtraOptions string, diskIdxCnt *int) []string {
+		if extraOptions != "" && extraOptions[0] != ',' {
+			extraOptions = "," + extraOptions
+		}
+		if deviceExtraOptions != "" && deviceExtraOptions[0] != ',' {
+			deviceExtraOptions = "," + deviceExtraOptions
+		}
+		qemuCmd = append(qemuCmd, "-drive", fmt.Sprintf("id=%s,file=%s,if=none,index=%d%s", id, file, *diskIdxCnt, extraOptions))
+		qemuCmd = append(qemuCmd, "-device", fmt.Sprintf("virtio-blk,drive=%s%s", id, deviceExtraOptions))
+		*diskIdxCnt += 1
+		return qemuCmd
+	}
```

Comment:
Would be nice to have this as a standalone function if possible (perhaps: return only the args that would be added, and then `qemuCmd = append(qemuCmd, qemuDiskArgs(..)...)` ?)

Reply:
It'd also be great to have this as a separate commit (and then merge via rebase onto main), or as a separate PR and have this one based on it. That way, it's clear which changes are refactoring(ish) vs adding the feature.

Reply:
sounds good I'll make it in a separate commit.